app/models/user.rb

```diff
@@ -30,6 +30,14 @@ def mentor?
     mentor_profile.present?
   end
 
+  def registered_mentor?
+    mentor_profile&.registration_completed?
+  end
+
+  def registered_early_career_teacher?
+    early_career_teacher_profile&.registration_completed?
+  end
+
```

Comment:
This feels like... duplication.

`registered_participant?` and use `participant_profile.registration_completed?`?